Update page template header. Remove duplicate file

ssrome opened the pull request:
Removed the duplicate file in the public folder and updated the one in the templates folder, with the council name in the header

<img width="1491" alt="template" src="https://user-images.githubusercontent.com/17614754/182784513-d229889b-a1b8-42cd-9d4d-8ff5967ff98f.png">

ssrome committed:
Update page template header. Remove duplicate file

Summary:

Tasks:

ssrome committed:
Update page template header. Remove duplicate file

Summary:

Tasks:

Joseaie commented on templates/default-page-template.html:
```diff
@@ -1,99 +1,173 @@
+
+    <script src="/javascripts/gov.js"></script>
+    <script>
+      window.GOVUKFrontend.initAll();
```
Would this have to be changed to what you said to James as well?

ssrome replied:
This is the change. There were two template files. The one I deleted had the one that needed changing

ssrome replied:
I've just realised what you might have been talking about with this. The while the filename is correct the reference to it isn't and it will be different to how to get to it, within the public folder. Not sure what to do about this, in this case

ssrome committed:
Update cypress config (#13)

Summary:

Tasks:

Porim committed:
Ft dob (#9)

* Create a date of birth page with some basic tests

Summary: ^^^

Tasks:

* forgoten tests...

* add some new lines

* fix bug with header

* remove cancel button and add back button

Porim committed:
Delete .DS_Store

removing file that shouldn't be on github

Porim committed:
Delete .DS_Store

removing file that shouldn't be on github

Joseaie reviewed (approved):
LGTM ✨

Joseaie committed:
Create 'Name Page' (#18)

Summary: This is the page where the users enter their first and last
names. I have tested the buttons, inputs and labels.

Tasks: The buttons need to be linked to the real pages; for now they are
just hashes (`#`)

ssrome committed:
Add extra readme instruction, for running tests (#14)

Summary: Add extra readme instruction, for running tests

ssrome committed:
Add header with service name (#16)

Summary: Updated the header according to design system, that includes styling for the service name

ssrome committed:
Update javascript path reference in template

ssrome committed:
Merge branch 'ft-update-template' into ft-update-template
